app/src/main/java/com/kabouzeid/gramophone/model/AbsCustomPlaylist.java

```diff
@@ -24,4 +28,18 @@ public AbsCustomPlaylist(Parcel in) {
 
     @NonNull
     public abstract ArrayList<Song> getSongs(Context context);
+
+    @NonNull
+    @Override
+    public String getInfoString(@NonNull Context context) {
+        String baseInfo = super.getInfoString(context);
+
+        int songCount = getSongs(context).size();
+        String songCountText = (songCount == 0) ?
+            context.getString(R.string.no_songs) :
+            String.valueOf(songCount) + " " + context.getString(R.string.songs);
```

Comment:
I don't quite remember, but is this how we're writing the # of songs in other places too? I would normally expect it to be a formatted string.

Reply:
Make sense. 
But this is the same method used elsewhere

https://github.com/kabouzeid/Phonograph/blob/8bbdfaa34a09e54280658767e5d12d1c9a6a1178/app/src/main/java/com/kabouzeid/gramophone/ui/fragments/player/AbsPlayerFragment.java#L135

https://github.com/kabouzeid/Phonograph/blob/8bbdfaa34a09e54280658767e5d12d1c9a6a1178/app/src/main/java/com/kabouzeid/gramophone/util/MusicUtil.java#L112

...

Reply:
I'll create a new PR to tackle all these, separately.